docs/Concepts/Privacy/PrivateAndPublic.md

```diff
@@ -1,43 +1,78 @@
 ---
-description: Transaction and Contract Privacy
+description: Public and Private Transactions
 ---
 
-# Transaction and Contract Privacy
+# Public and Private Transactions
 
-GoQuorum achieves Transaction Privacy by:
+GoQuorum achieves transaction privacy by:
 
-1. Enabling transaction Senders to create a private transaction by marking who is privy to that transaction via the `privateFor` parameter
-1. Replacing the payload of a private transaction with a hash of the encrypted payload, such that the original payload is not visible to participants who are not privy to the transaction
-1. Storing encrypted private data off-chain in a separate component called the [Privacy Manager](PrivateTransactionManager.md). The Privacy Manager encrypts private data, distributes the encrypted data to other parties that are privy to the transaction, and returns the decrypted payload to those parties
+1. Enabling transaction senders to create private transactions by marking who is privy to a transaction via the
+  `privateFor` parameter.
 
-GoQuorum introduces the notion of 'Public Transactions' and 'Private Transactions'. Note that this is a notional concept only and GoQuorum does not introduce new Transaction Types, but rather, the Ethereum Transaction Model has been extended to include an optional `privateFor` parameter (the population of which results in a Transaction being treated as private by GoQuorum) and the Transaction Type has a new `IsPrivate` method to identify such Transactions.
+1. Storing encrypted private data off-chain in a separate component called the
+  [private transaction manager](PrivateTransactionManager.md).
+  The private transaction manager encrypts private data, distributes the encrypted data to other parties that are privy
+  to the transaction, and returns the decrypted payload to those parties.
 
-## Public Transactions
+1. Replacing the payload of a private transaction with a hash of the encrypted payload, such that the original payload
+   is not visible to participants who are not privy to the transaction.
 
-Public Transactions are those Transactions whose payload is visible to all participants of the same GoQuorum network. These are [created as standard Ethereum Transactions in the usual way](https://github.com/ethereum/wiki/wiki/JavaScript-API#web3ethsendtransaction).
+!!! note
 
-Examples of Public Transactions may include Market Data updates from some service provider, or some reference data update such as a correction to a Bond Security definition.
+    While GoQuorum introduces the notion of "public transactions" and "private transactions," it does not introduce new
+    transaction types.
+    Rather, it extends the Ethereum transaction model to include an optional `privateFor` parameter (whose use results
```

Comment:
Have trouble understanding "(whose use results in GoQuorum treating a transaction as private)"